saraGuess.js

```diff
@@ -0,0 +1,53 @@
+const readline = require('readline');
+const rl = readline.createInterface(process.stdin, process.stdout);
+
+function ask(questionText) {
+  return new Promise((resolve, reject) => {
+    rl.question(questionText, resolve);
+  });
+}
+
+start();
+
+async function start() {
+  // welcome the user and wait to finish
+  const answer =await welcome()
+  let guess = getRandomInt(1, 101)
```

Comment:
This is a good way to fix the off by 1 bug where getRandomInt is exclusive on the high end. You could have also added 1 to the range in the function's definition.